### PR TITLE
Load pretrained BERT models when using classifier

### PR DESCRIPTION
**Pull request by MetcalfeTom**
Happy Easter,

Changes:
- `BertIntentClassifier` now allows a pretrained model to be loaded by specifying `pretrained_model_dir` in the config YAML
- Tensorflow graph was refactored to suppress dropout when predicting
- Two-sequence input features now enabled
- Code formatting 💜

Also includes the fix from https://github.com/Revmaker/innatis/pull/17 so that models can be used to process messages

### CODE REVIEW

File: innatis/classifiers/bert_intent_classifier.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def process(self, message: Message, **kwargs: Any) -> None:
         probabilities = list(result["probabilities"][0])
 
         with self.session.as_default():
-            index = tf.argmax(probabilities, axis=0).eval()
+            index = tf.argmax(probabilities, axis=0).eval(session=tf.Session())
```
Comment (Ghostvv) [inline innatis/classifiers/bert_intent_classifier.py]: I'm not sure this is a correct thing to do, as far as I understand in this case the tensor will be evaluated in another session and not self.session

Comment (MetcalfeTom) [reply innatis/classifiers/bert_intent_classifier.py]: Yes I thought so too, but my intuition now is that `with self.session.as_default()` writes `self.session` as the default graph whenever a `tf.Session()` is initialised.

I ran some evaluations with `rasa.nlu.test` on a finetuned model vs one with no finetuning (both running this processing code) and the f1 score differed between the two by almost 0.9.  So if this is not the case then I can't explain why

**Comment by samhavens** [comment]
Sorry for being MIA! 😬
@MetcalfeTom @Ghostvv  @johnnyworker1012 THANK YOU!

How can I help? I'm just starting to look everything over now

**Comment by MetcalfeTom** [comment]
Hi @samhavens,

This PR mainly includes a couple refactoring changes such that the standard `rasa_nlu` toolkit works with it (e.g. evaluating and message processing).  I have been testing its functionality - when used as a server, message processing time is, expectedly, a little slower than the standard pipelines.  It's good to merge from my end

**Comment by samhavens** [comment]
@MetcalfeTom - this should be automated... but would you mind bumping the version to `0.6.0` https://github.com/MetcalfeTom/innatis/blob/bert-processing/setup.py#L20 ?

**Comment by samhavens** [comment]
@blakeandrewwood @akingbadeayobami @nimf 
I installed this locally just to see if any of the changes broke our current Rasa/Innatis pipeline. I trained a YBIR agent, and it completed:
```sh
...
2019-05-11 16:17:42 INFO     rasa_nlu.classifiers.embedding_intent_classifier  - Finished training embedding classifier, loss=0.061, train accuracy=0.994
2019-05-11 16:17:42 INFO     rasa_nlu.model  - Finished training component.
2019-05-11 16:17:43 INFO     rasa_nlu.model  - Successfully saved model into '/Users/samhavens/ybir-bot-agent/models/ybir-shopbot/core'
...
2019-05-11 16:21:34 INFO     rasa_nlu.classifiers.embedding_intent_classifier  - Finished training embedding classifier, loss=0.011, train accuracy=0.997
2019-05-11 16:21:34 INFO     rasa_nlu.model  - Finished training component.
2019-05-11 16:21:35 INFO     rasa_nlu.model  - Successfully saved model into '/Users/samhavens/ybir-bot-agent/models/ybir-shopbot/meta'
...
2019-05-11 16:22:30 INFO     rasa_nlu.classifiers.embedding_intent_classifier  - Finished training embedding classifier, loss=0.214, train accuracy=0.967
2019-05-11 16:22:30 INFO     rasa_nlu.model  - Finished training component.
2019-05-11 16:22:30 INFO     rasa_nlu.model  - Successfully saved model into '/Users/samhavens/ybir-bot-agent/models/ybir-shopbot/smalltalk'
```

**Comment by samhavens** [comment]
I'm still testing using BERT with the YBIR agent, but since this doesn't break anything for CarLabs, I think we should merge it in.

**Comment by MetcalfeTom** [comment]
Bumped the version number.  Pleasure to contribute 🚀
